notebooks/week2/05.log_and_register_fe_model.py

```diff
@@ -175,8 +174,4 @@
 mlflow.register_model(
     model_uri=f'runs:/{run_id}/lightgbm-pipeline-model-fe',
     name=f"{catalog_name}.{schema_name}.power_consumptions_model_fe")
```

Comment:
have you also tried to serve this model?

you have a nice example with pyfunc model - that's enough for week3, I'm just asking for curiosity :)

Reply:
hey, no I did not try because of time limitations